Cargo.toml

```diff
@@ -29,12 +29,12 @@ required-features = ["ogg"]
 [dependencies]
 byteorder = "1.0"
 tinyvec = { version = "1.0", features = ["alloc"] }
-ogg = { version = "0.8", optional = true }
+ogg = { version = "0.8", optional = true, path = "../ogg/" }
```

Comment:
Wow what happened here

Reply:
So it seems like you have a local edition of the `ogg` crate, which I can't find anywhere and thus can't build. :( Mind at least pushing the fork somewhere?

Reply:
I'm patching `ogg` crate at RustAudio/ogg#18 and RustAudio/ogg#22 as well, and since this PR depends on those PRs, this temporary measure is taken to handle this.  We can remove it after they are merged.